source/reference/api/start.txt

```diff
@@ -111,16 +111,38 @@ Request Body Parameters
 
        Default value is ``false``.
 
+   * - ``includeNamespaces``
+     - array
+     - Optional
+     - Filters the databases or collections to sync. The filter document
```

Comment:
The filter json (document might be confusing)

Reply:
Done